lms/djangoapps/bookmarks/serializers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Serializers for Bookmarks.
+"""
+from rest_framework import serializers
+
+from .models import Bookmark
+
+
+class BookmarkSerializer(serializers.ModelSerializer):
+    """
+    Class that serializes the Bookmark model.
+    """
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        try:
+            fields = kwargs['context'].pop('fields', [])
+        except KeyError:
+            from .views import DEFAULT_FIELDS
+            fields = DEFAULT_FIELDS
+        # Instantiate the superclass normally
+        super(BookmarkSerializer, self).__init__(*args, **kwargs)
+
+        if fields:
+            # Drop any fields that are not specified in the `fields` argument.
+            required_fields = set(fields)
+            all_fields = set(self.fields.keys())
+            for field_name in all_fields - required_fields:
+                self.fields.pop(field_name)
+
+    id = serializers.SerializerMethodField('resource_id')  # pylint: disable=invalid-name
```

Comment:
@cahrens Actually the field names in Model are different than the names in response. e.g. the `course_key` in models would be `course_id` and `usage_key` would be `usage_id`. That's the reason to specify these serializers explicitly, but yes we can skip the `path` from here.